pages/common/chmod.md

```diff
@@ -1,36 +1,24 @@
 # chmod
 
-> Change the access permissions of a file or directory.
+> Change file/directory access permissions.
 > More information: <https://www.gnu.org/software/coreutils/chmod>.
 
-- Give the [u]ser who owns a file the right to e[x]ecute it:
+- Add [r]ead/[w]rite/e[x]ecute permissions for a file/directory owner:
 
-`chmod u+x {{file}}`
+`chmod +{{r|w|x}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
 
-- Give the [u]ser rights to [r]ead and [w]rite to a file/directory:
+- Add specific permissions for an owner/users in [g]roup/[o]ther users/[a]ll at once:
 
-`chmod u+rw {{file_or_directory}}`
+`chmod {{u|g|o|a}}+{{r}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
 
-- Remove e[x]ecutable rights from the [g]roup:
+- Add/remove/replace specific permissions:
 
-`chmod g-x {{file}}`
+`chmod {{u}}{{+|-|=}}{{r}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
 
-- Give [a]ll users rights to [r]ead and e[x]ecute:
+- Remove all permissions:
 
-`chmod a+rx {{file}}`
+`chmod {{u}}= {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
 
-- Give [o]thers (not in the file owner's group) the same rights as the [g]roup:
+- Change permissions recursively:
 
-`chmod o=g {{file}}`
-
-- Remove all rights from [o]thers:
-
-`chmod o= {{file}}`
-
-- Change permissions recursively giving [g]roup and [o]thers the ability to [w]rite:
-
-`chmod -R g+w,o+w {{directory}}`
-
-- Recursively give [a]ll users [r]ead permissions to files and e[X]ecute permissions to sub-directories within a directory:
-
-`chmod -R a+rX {{directory}}`
+`chmod --recursive {{u}}{{+}}{{r}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
```

Comment:
Using long form options is good, but in this instance I feel since this is a frequently used command like `ls` we should use short options to reduce typing.

Reply:
I think it's better to split page here into two ones: with short options and long options.

Reply:
We don't currently have provision for multiple versions of pages like that. Let's use short options on this page, as `chmod` is often used on a regular basis, similar to `ls`.

To be more specific here, ref the guidelines we don't necessarily need to use long options *all* the time. See also the `tar` command's page for further inspiration of how short options can be demonstrated effectively.

Reply:
But there are already [several PRs](https://github.com/tldr-pages/tldr/pulls/EmilySeville7cfg) doing so with `add` and `refresh` words in the titles. It was not my idea to split pages with long and short options, @pixelcmtd suggested it. But I like it. I am planning to refresh contribution guidelines and explicitly write this rule. There are just general pages which can use both long and short options interchangeably.

Reply:
I'm not sure what having `add` and `refresh` in the title of a PR has anything to do with this issue.

I'm not even sure what you mean by "split pages with long and short options", or how it would even be implemented without either:

1. Breaking clients
2. Causing confusion
3. Resulting in the additional extra work of maintaining multiple copies of a page.

Reply:
> I'm not even sure what you mean by "split pages with long and short options", or how it would even be implemented without either:
>
>   1. Breaking clients
>   1.  Causing confusion
>   1.  Resulting in the additional extra work of maintaining multiple copies of a page.

When clients can be confused? Current specification mandates clients to have --platform option to point to a specific platform with a particular page.

Yeah, it is an extra work now. We can solve this issue with adding support for switching between long and short options via smth like this: `tldr --long-options` and pages with such placeholders:

```
command {{{-s|--long}}}
```

I have no idea about what syntax is the best. We have to discuss it.

Reply:
> Using long form options is good, but in this instance I feel since this is a frequently used command like ls we should use short options to reduce typing.

Style guide now [requires](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax) long portable options, and it's possible to duplicate pages to show their different options's counterparts, it's valid. Issue is resolved.